site/content/docs/2.0/extend/icons.md

```diff
@@ -17,9 +17,174 @@ Quickstart, and Arizona Bootstrap.
 - [Arizona Icons](https://github.com/az-digital/az-icons) (Arizona branded icons)
 - [Google Material icons (Sharp)](https://material.io/resources/icons/?style=sharp)
 
+## Arizona Icons Implementation
+
+Include a reference to the Arizona Icons' CDN to your project in order to use the font icons:
+
+```html
+<link href="https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css rel="stylesheet">
+```
+
+[Refer to Arizona Icons GitHub repository](https://github.com/az-digital/az-icons) for more information and resources.
+
+<div class="row">
+  <div class="col-4 col-md-3 col-sm-1">
+    <div class="well text-center mb-4">
```

Comment:
Just realized this during your demo. Wells have actually been deprecated in Bootstrap 4 so for consistency it would be good to swap out `.well` for `.card bg-gray-200`. Just to avoid using deprecated classes in the docs.
```suggestion
    <div class="card bg-gray-200 text-center mb-4">
```